README.md

```diff
@@ -781,7 +785,11 @@ Response:
 		"title": "decompress/unpack",
 		"app_weight": 1,
 		"file_type": "CompressedFileFormatArchive",
-	}]]
+	}]],
+    "fileinfo": [
+        {"prefix": "file1.txt", "suffix": null},
+        {"prefix": "file", "suffix": "zip"}
```

Comment:
I wonder if the FE would actually prefer a mapping of mappings instead.

Reply:
What I really wanted to do was to add the `fileinfo` to the `mappings` key, but since that's a list of lists I couldn't without breaking backwards compatibility.  As such, I made `fileinfo` mirror the `mappings` key so that they could be iterated through at the same time with an index or zipping them together. I still feel like that's probably the best way to go.

Although now the `fileinfo` key exists it might make more sense to move the `filetype` from `mappings` to `fileinfo`.

Actually - if `fileinfo` was a mapping what would the key be?